sstables/sstables.cc

```diff
@@ -1971,7 +1971,7 @@ void maybe_add_summary_entry(summary& s, const dht::token& token, bytes_view key
         state.next_data_offset_to_write_summary += state.summary_byte_cost * entry_size;
         s.add_summary_data(token.data());
         auto key_data = s.add_summary_data(key);
-        s.entries.push_back({ dht::token(dht::token::kind::key, token._data), key_data, index_offset });
+        s.entries.push_back({ token, key_data, index_offset });
     }
```

Comment:
Here too.

Reply:
What do you want me to do here @avikivity?

Reply:
I moved it out to a separate patch.